neurobooth_os/transfer_data.py

```diff
@@ -0,0 +1,66 @@
+"""
+    Moves data from local storage to network storage
+"""
+import os
+from subprocess import PIPE, Popen, STDOUT, CalledProcessError
+import argparse
+
+from neurobooth_os import config
+from neurobooth_os.util.constants import NODE_NAMES
+from neurobooth_os.log_manager import make_default_logger
+
+
+def log_output(logger, pipe):
+    for line in iter(pipe.readline, b''):  # b'\n'-separated lines
+        logger.info(str(line, "utf-8").strip('\r\n'))
+
+
+def main(args: argparse.Namespace):
+
+    logger = make_default_logger()
+
+    destination = config.neurobooth_config["remote_data_dir"]
+
+    source = config.neurobooth_config[args.source_node_name]["local_data_dir"]
```

Comment:
Where do we map from the shorthand node names (e.g., CTL) to the ones in the config (e.g., 'control')?

Reply:
in the config file. I'm not sure why we need both

Reply:
Not sure either... but if this script runs then I was just missing something small